meson.build

```diff
@@ -79,23 +79,14 @@ else
   pkg_prefix = option_pkg_prefix
 endif
 
-rootprefix = get_option('rootprefix')
-if rootprefix == ''
-rootprefix = '/'
-endif
-
-bindir = rootprefix / get_option('bindir')
-libdir = rootprefix / get_option('libdir')
+bindir = get_option('bindir')
+libdir = get_option('libdir')
 libexecdir = get_option('libexecdir')
-if os == 'Linux' and libexecdir == 'libexec'
-  libexecdir = 'lib'
-endif
-libexecdir = rootprefix / libexecdir
-rc_libexecdir = libexecdir / 'rc'
+sbindir = get_option('sbindir')
+rc_libexecdir = get_option('prefix') / libexecdir / 'rc'
```

Comment:
should we consider changing `rc` to `openrc`, or maybe making that an `rc_dir` build option?

Reply:
I'm open to changing this to openrc, probably in another pr, but I don't think we should make this configurable. The FHS says that the directory under libexec should be the program name.

Reply:
fair, let's leave it for a follow up